homeassistant/components/wwlln/config_flow.py

```diff
@@ -46,38 +35,49 @@ class WWLLNFlowHandler(config_entries.ConfigFlow):
             }
         )
 
+    async def _show_form(self, errors=None):
+        """Show the form to the user."""
         return self.async_show_form(
-            step_id="user", data_schema=data_schema, errors=errors or {}
+            step_id="user", data_schema=self.data_schema, errors=errors or {}
         )
 
     async def async_step_import(self, import_config):
         """Import a config entry from configuration.yaml."""
+        default_window_seconds = DEFAULT_WINDOW.total_seconds()
+        if (
+            CONF_WINDOW in import_config
+            and import_config[CONF_WINDOW] < default_window_seconds
+        ):
+            LOGGER.error(
+                "Refusing to use too-small window (%s < %s)",
+                import_config[CONF_WINDOW],
+                default_window_seconds,
+            )
+            return self.async_abort(reason="window_too_small")
```

Comment:
We could validate this instead in the yaml config schema with `vol.Range`.

Reply:
https://github.com/home-assistant/core/pull/32621